test/e2e/crd_e2e_test.go

```diff
@@ -89,7 +89,7 @@ var _ = Describe("CRD Versions", func() {
 		installPlanName := subscription.Status.InstallPlanRef.Name
 
 		// Wait for InstallPlan to be status: Complete before checking resource presence
-		fetchedInstallPlan, err := fetchInstallPlan(GinkgoT(), crc, installPlanName, buildInstallPlanPhaseCheckFunc(operatorsv1alpha1.InstallPlanPhaseComplete))
+		fetchedInstallPlan, err := fetchInstallPlan(GinkgoT(), crc, testNamespace, installPlanName, buildInstallPlanPhaseCheckFunc(operatorsv1alpha1.InstallPlanPhaseComplete))
```

Comment:
change `installPlanName := subscription.Status.InstallPlanRef.Name` -> `installPlanRef := subscription.Status.InstallPlanRef` and then update the fetchInstallPlan function to use the installPlanRef.Name and installPlanRef.Namespace values as arguments?

Reply:
Same comment throughout all of the crd_e2e_test.go changes that are present in this PR - we're still using the `testNamespace` global variable which will still contribute to testing pollution over time.

Reply:
nnooooooooooooooooooooo

Reply:
I tried to make the changes to the install plan name stuff, but we don't always have a reference. Let's keep the scope of the changes as much as possible just to the namespace stuff. We can pretty stuff up later.

Reply:
Yep that's fine with me :+1: